.github/workflows/main.yml

```diff
@@ -64,6 +64,7 @@ jobs:
           --health-timeout 5s
         ports:
           - 5432:5432
+        command: ["-c", "fsync=off"]
```

Comment:
Ah ? I thought you said this didn't change anything ?

Reply:
Ah, just saw the table.

Reply:
It's not a huge win, but it looks like we do save a few seconds for the entire test suite.